Gemfile

```diff
@@ -15,34 +15,34 @@ gem 'doorkeeper', '~> 5.2'
 gem 'doorkeeper-i18n', '~> 5.2'
 gem 'exifr', '~> 1.3'
 gem 'factory_bot_rails', '~> 5.0'
-gem 'faker', '~> 1.9'
-gem 'friendly_id', '~> 5.2.0'
+gem 'faker', '~> 2.8'
+gem 'friendly_id', '~> 5.2'
 gem 'http', '~> 4.0'
 gem 'iban-tools', '~> 1.1.0'
 gem 'icalendar', '~> 2.4'
-gem 'jsonapi-authorization', '~> 3.0.1'
+gem 'jsonapi-authorization', '~> 3.0'
 gem 'jsonapi-resources', '~> 0.9.1.beta2'
 gem 'mail', '~> 2.6'
-gem 'mailgun-ruby', '~> 1.1.11'
+gem 'mailgun-ruby', '~> 1.2'
 gem 'message_bus', '~> 2.0'
 gem 'mini_magick', '~> 4.6'
 gem 'paper_trail', '~> 10.0'
 gem 'paranoia', '~> 2.2'
 gem 'pg', '~> 1.0'
 gem 'phonelib'
-gem 'puma', '~> 3.7'
+gem 'puma', '~> 4.3'
 gem 'pundit', '~> 2.0'
 gem 'rack-attack', '~> 6.0'
 gem 'rack-cors', '~> 1.0', require: 'rack/cors'
 gem 'rails', '~> 6.0'
 gem 'rails-i18n', '~> 6.0'
-gem 'redis-rails', '~> 5.0.1'
-gem 'rqrcode', '~> 0.10'
+gem 'redis', '~> 4.0'
```

Comment:
Ik zie dat we van redis-rails naar gewoon redis gaan. Is de functionaliteit gelijk gebleven?

Reply:
Dat hoop ik, tutorials online zeggen van wel (we gebruiken nu de vanilla rails cache store ipv een losse gem)